src/main/java/utils/driverUtils/DriverWrapper.java

```diff
@@ -123,5 +125,15 @@ public void quiteDriver() {
         }
         driver = null;
     }
+
+    public void saveScreenshot(String testName) throws IOException {
```

Comment:
Please, do not throw exception here. Create try/catch block inside the method and process IOException